test/parallel/test-http-response-close-event-race.js

```diff
@@ -0,0 +1,88 @@
+var common = require('../common');
+var assert = require('assert');
+var http = require('http');
+
+var clientRequest = null;
+var serverResponseFinishedOrClosed = 0;
+var testTickCount = 3;
+
+var server = http.createServer(function (req, res) {
+  console.log('server: request');
```

Comment:
Of course I could remove them - I kept with the other tests which also contain `console.log`s (output is hidden anyway in normal test run).
I thought, especially race conditions are much more understandable with some debugging output - also for future refactoring/debugging.